Document JetBrains plugin support for Federation (#2382)
apollographql/federation: docs/source/building-supergraphs/jetbrains-ide-support.mdx

> # JetBrains IDE Support

Reviewer: ```suggestion
---
title: Federation support in JetBrains IDEs
---
```

> The [GraphQL Plugin for JetBrains](https://plugins.jetbrains.com/plugin/8097-graphql/) supports Apollo Federation, but it is disabled by default. To enable it:

Reviewer: ```suggestion
The [GraphQL Plugin for JetBrains](https://plugins.jetbrains.com/plugin/8097-graphql/) provides federation-specific development features, such as ___ and ___.

To enable these features in your JetBrains IDE, do the following after installing the plugin:
```

> 1. Open the "Settings" window
> 2. Expand the "Languages & Frameworks" section
> 3. Click on "GraphQL"
> 4. Check the "Federation" checkbox

Reviewer: ```suggestion
1. Open your IDE's **Settings** window.
2. Expand the **Languages & Frameworks** section.
3. Click **GraphQL**.
4. Check the **Federation** checkbox.
```

<img
    class="screenshot"
    alt="An animation showing the process of enabling Apollo Federation support in the GraphQL plugin for JetBrains (described above)"
    src="../images/JetBrains-plugin.webp"
    width="600"
/>

> This plugin supports all the IntelliJ-based IDEs, such as:

Reviewer: ```suggestion
This plugin supports all IntelliJ-based IDEs, including:
```

---
title: Federation support in JetBrains IDEs
---

> The [GraphQL Plugin for JetBrains](https://plugins.jetbrains.com/plugin/8097-graphql/) provides federation-specific development features, such as ___ and ___. **However**, you must _enable_ this federation support after installing the plugin. Otherwise, your IDE might display unexpected errors while you're working with a subgraph schema.

Author: ```suggestion
The [GraphQL Plugin for JetBrains](https://plugins.jetbrains.com/plugin/8097-graphql/) provides federation-specific development features, such as auto-complete for federation directives. **However**, you must _enable_ this federation support after installing the plugin. Otherwise, your IDE might display unexpected errors while you're working with a subgraph schema.
```
